rtgui/history.cc

```diff
@@ -155,8 +155,10 @@ History::History (bool bookmarkSupport) : blistener(NULL), tpc (NULL), bmnum (1)
 void History::initHistory ()
 {
 
+    selchangehist.block(true);
```

Comment:
Ah, I missed that change. I'll use it then and will commit, since the fix is trivial and no one complain.